cli/src/semgrep/semgrep_main.py

```diff
@@ -52,7 +52,13 @@
 
 def get_file_ignore() -> FileIgnore:
     TEMPLATES_DIR = Path(__file__).parent / "templates"
-    workdir = Path.cwd()
+    try:
+        workdir = Path.cwd()
+    except FileNotFoundError:
+        workdir = Path.home()
+        logger.warn(
+            f"Current working directory does not exist! Using {workdir} as working directory"
```

Comment:
Maybe clarify that we are looking for .semgrepignore in workdir not that we are scanning workdir which is what this might sound like

Reply:
Yea makes sense